src/cleanvision/imagelab.py

```diff
@@ -567,21 +603,35 @@ def visualize(
                 raise ValueError("issue_types list is empty")
             for issue_type in issue_types:
                 self._visualize(issue_type, num_images, cell_size)
+        elif image_files:
+            # todo: write test
+            if len(image_files) == 0:
+                raise ValueError("image_files list is empty.")
+            images = [Image.open(path) for path in image_files]
+            titles = [path.split("/")[-1] for path in image_files]
+            VizManager.individual_images(
+                images,
+                titles,
+                ncols=self._config["visualize_num_images_per_row"],
+                cell_size=cell_size,
+            )
```

Comment:
this block seems copy/pasted a lot, consider consolidating to have it only be one place in the code (ignore this suggestion if it's nontrivial)

Reply:
This is just a long method call and consolidating right now is not trivial. If we have a Reporter class in future, then it's worth refactoring the code.